apps/etterlatte-grunnlag/src/main/kotlin/grunnlag/GrunnlagService.kt

```diff
@@ -249,7 +276,7 @@ class RealGrunnlagService(
         return Grunnlagsopplysning(
             id = UUID.randomUUID(),
             kilde =
-                if (this.innsender!! == Vedtaksloesning.PESYS.name) {
+                if (this.innsender == Vedtaksloesning.PESYS.name) {
                     Grunnlagsopplysning.Pesys.create()
                 } else {
                     Grunnlagsopplysning.Privatperson(this.innsender!!, Tidspunkt.now())
```

Comment:
Burde !! bort herifrå også?

Reply:
Innsender er potensielt `null` der, så da trenger vi litt mer logikk rundt det. 
Fungerer på den øverste siden man da bare sammenligner to `String?` og da var `!!` helt unødvendig